src/sensors.cpp

```diff
@@ -326,6 +334,18 @@ bool Sensors::update_imu(void)
   }
 }
 
+
+void Sensors::get_filtered_IMU_(turbomath::Vector &accel, turbomath::Vector &gyro, uint64_t &stamp_us)
```

Comment:
This setup assumes that there's only one consumer of the filtered IMU (which is true right now). Would it be worth trying to remove that assumption at some point?

Reply:
Maybe.  In the future, if we move to a Kalman filter, we may need to preintegrate attitude, which would require a refactoring of IMU anyway.